assets/js/components/notifications/AdBlockingRecoverySetupSuccessBannerNotification.js

```diff
@@ -75,7 +83,11 @@ export default function AdBlockingRecoverySetupSuccessBannerNotification() {
 			`${ viewContext }_adsense-abr-success-notification`,
 			'view_notification'
 		);
-	}, [ viewContext ] );
+
+		if ( usingProxy ) {
+			triggerSurvey( 'abr_setup_completed', { ttl: DAY_IN_SECONDS } );
```

Comment:
The AC doesn't call for this ttl since it's intended to be an on-demand action but the implementation is more of a "view" even if it will only be triggered in response to the setup completion 👍